.github/workflows/on_push_branch__execute_ci_cd.yml

```diff
@@ -82,12 +82,12 @@ jobs:
             _build
           key: ${{ runner.os }}-mix-v4-${{ hashFiles('**/mix.lock') }}
           restore-keys: ${{ runner.os }}-mix-v4
-      
+
```

Comment:
sorry for those drive-bys, I can get them out but I have my editor set to trim unnecessary whitespace and ensure line break at the end of the file :)